mteb/tasks/Classification/metagene/HumanMicrobiomeProjectDemonstrationClassification.py

```diff
@@ -0,0 +1,254 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskClassification import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class HumanMicrobiomeProjectDemonstrationClassificationDisease(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="HumanMicrobiomeProjectDemonstrationClassificationDisease",
+        description="A Classification task for predicting the disease from the Human Microbiome Project Demonstration dataset.",
+        reference="https://huggingface.co/datasets/metagene-ai/HumanMicrobiomeProjectDemonstration/tree/main/hmpd/disease",
+        dataset={
+            "path": "metagene-ai/HumanMicrobiomeProjectDemonstration",
+            "name": "disease",
+            "revision": "main",
+        },
+        type="Classification",
+        category="s2s",
+        modalities=["text"],
```

Comment:
A text modality seems someone misleadering. It input is a string of base pairs right?

Reply:
keeping "text" here might be fine but it should be clear for the description. I would also make it clear in the task subtypes.